packages/boa/package.json

```diff
@@ -5,14 +5,15 @@
   "main": "lib/index.js",
   "scripts": {
     "clean": "sh tools/clean-python.sh",
-    "preinstall": "node tools/install-python.js && node tools/install-requirements.js",
-    "postinstall": "npm run build",
+    "preinstall": "make -C ./pybind11/ && node tools/install-python.js && node tools/install-requirements.js",
+    "postinstall": "npm run compile",
```

Comment:
We need to keep this using build script because both they are used in installing via NPM, however, the `compile` script is used with `lerna run compile`.

Reply:
Done.